ecosystem/typescript/sdk/examples/typescript/http2-fetch-h2/index.ts

```diff
@@ -0,0 +1,268 @@
+import { AptosAccount, AptosClient, BCS, HexString, TxnBuilderTypes } from "../../../dist";
+import { Timer } from "timer-node";
+import { fetch } from "fetch-h2";
+import { exit } from "process";
+
+// const FULLNODE_URL = "http://0.0.0.0:8080/v1";
+// const FAUCET_URL = "http://0.0.0.0:8081";
+
+const FULLNODE_URL = "https://fullnode.testnet.aptoslabs.com/v1";
+const FAUCET_URL = "https://faucet.testnet.aptoslabs.com";
+
+async function main() {
+  const timer = new Timer();
+
+  const accountsCount = 5; // creates 400 accounts in total
+  const firstPass = 100;
+  const readAmplification = 100; // tests 200 accounts * 3 = 600 get calls
+  let accountSequenceNumber: AccountSequenceNumbers | null = null;
+
+  console.log("starting...");
+  timer.start();
+  // create accounts
+  const accounts: AptosAccount[] = [];
+  const recipients: AptosAccount[] = [];
+  for (let i = 0; i < accountsCount; i++) {
+    accounts.push(new AptosAccount());
+    recipients.push(new AptosAccount());
+  }
+  console.log(`${accounts.length * 2} accounts created`);
```

Comment:
```suggestion
  console.log(`${accounts.length + recipients.length} accounts created`);
```

nit: Number still the same, but i felt like its more read-able this way :D